### ISSUE TITLE
[BUG] SWAT's missing voiceover

### ISSUE THREAD

**Issue by semodo** [eezstreet/SWATEliteForce#407]
SWAT officers are supposed to say something while they are cuffing npc's, and after they cuffed them, they report them to TOC. While officers are cuffing hostages they say things like "Relax, everything will be ok" and after they report them. There is a same mechanic for suspects, but officers are not saying anything while cuffing them, they only report them. It isn't missing only in v7, but in SEF in general.
